.github/workflows/backport-assistant.yml

```diff
@@ -41,3 +42,15 @@ jobs:
             -X POST \
             -d "{ \"body\": \"${github_message}\"}" \
             "https://api.github.com/repos/${GITHUB_REPOSITORY}/issues/${{ github.event.pull_request.number }}/comments"
+  backport-ent:
```

Comment:
I didn't check for failure for `backport-ent` job since this job only sends the payload over to ent, but now that I think about it, it might be a good idea to check for failure for this job as well. So if `backport-ent` job fails, the failure would be commented on the CE PR.

Reply:
Yeah, I could imagine we'd basically see this fail when the GH token expires or GH drops the ball. Probably not too frequent, but doesn't hurt to cover